docker/hoodie/hadoop/hive_base/pom.xml

```diff
@@ -57,10 +57,10 @@
               <tasks>
                 <copy file="${project.basedir}/../../../../packaging/hudi-hadoop-mr-bundle/target/hudi-hadoop-mr-bundle-${project.version}.jar" tofile="target/hoodie-hadoop-mr-bundle.jar" />
                 <copy file="${project.basedir}/../../../../packaging/hudi-hive-bundle/target/hudi-hive-bundle-${project.version}.jar" tofile="target/hoodie-hive-bundle.jar" />
-                <copy file="${project.basedir}/../../../../packaging/hudi-spark-bundle/target/hudi-spark-bundle-${project.version}.jar" tofile="target/hoodie-spark-bundle.jar" />
+                <copy file="${project.basedir}/../../../../packaging/hudi-spark-bundle/target/hudi-spark-bundle_${scala.binary.version}-${project.version}.jar" tofile="target/hoodie-spark-bundle_${scala.binary.version}.jar" />
```

Comment:
@zhedoubushishi : You are on right track. Don't add scala version to the target jars inside docker. The docker setup has hard-wired this configuration. Removing the version from "tofile" entry should hopefully have the tests pass.

Reply:
> @zhedoubushishi : You are on right track. Don't add scala version to the target jars inside docker. The docker setup has hard-wired this configuration. Removing the version from "tofile" entry should hopefully have the tests pass.

Thanks for your advise. I'll have a try.